jenkins/ci.opensuse.org/openstack-repocheck.yaml

```diff
@@ -39,8 +40,8 @@
     execution-strategy:
       combination-filter: |
        !(["Cloud:OpenStack:Master"].contains(project) && ["SLE_12_SP2"].contains(repository) ||
```

Comment:
Leap15 is now also enabled for master? Otherwise it needs to be excluded here

Reply:
I would like to have it enabled for now